formatter/formatter.go

```diff
@@ -0,0 +1,32 @@
+package formatter
+
+import (
+	"bytes"
+	"encoding/json"
+	"golang.org/x/xerrors"
+	"os"
+	"text/template"
+)
+
+func Format(tpl string, inputData []byte, outputFile string) error {
```

Comment:
By default, we use `githubTpl` from `templates.go`.

Can we use this default template and add `WithTemplate` function for tests?

Reply:
same for mdFile